__testUtils/mockAccessControl.js

```diff
@@ -26,12 +26,12 @@ export default function mockAccessControl(
     deleteFile: jest.fn().mockResolvedValue({ response: {} }),
     getPermissions: jest.fn().mockResolvedValue(permissions),
     savePermissionsForAgent: jest.fn().mockResolvedValue({ response: {} }),
-    addAgentToNamedPolicy: jest.fn().mockResolvedValue({ response: {} }),
-    addAgentToCustomPolicy: jest.fn().mockResolvedValue({ response: {} }),
-    getPermissionsForNamedPolicies: jest.fn().mockResolvedValue(permissions),
-    getPermissionsForCustomPolicies: jest.fn().mockResolvedValue(permissions),
-    removeAgentFromNamedPolicy: jest.fn().mockResolvedValue({ response: {} }),
-    removeAgentFromCustomPolicy: jest.fn().mockResolvedValue({ response: {} }),
+    addAgentToPolicy: jest.fn().mockResolvedValue({ response: {} }),
+    // addAgentToCustomPolicy: jest.fn().mockResolvedValue({ response: {} }),
+    getPermissionsForPolicy: jest.fn().mockResolvedValue(permissions),
+    // getPermissionsForCustomPolicies: jest.fn().mockResolvedValue(permissions),
+    removeAgentFromPolicy: jest.fn().mockResolvedValue({ response: {} }),
+    // removeAgentFromCustomPolicy: jest.fn().mockResolvedValue({ response: {} }),
```

Comment:
Rather than being commented, should these be removed?

Reply:
Yes, my bad, forgot to remove this when cleaning up code (initially didn't know whether they should be removed, which is why I only commented them out in the first place)